src/pages/explainer page

```diff
@@ -0,0 +1,33 @@
+# How Open Source Can Help You Save Money and Time?
+
+
+## What is Open Source?
+
+Open source is products and tools containing source code and designs that the public can modify and change. Think of them as Google Slides or Microsoft PowerPoint presentations you create with your team for business meetings, but for software engineers, backend engineers, frontend developers, and other folks from different industries.
```

Comment:
[Proofreading] Hmm, I don't think this phrasing is what we'd want to put forward.

* The biggest win around open source for many people is that they can _view_ the source code. Telling people that the advantages of open source revolve around _modifying_ (or otherwise taking any active action) could give them the impression using open source requires taking up dev time. Which is not the case.
    * Suggestion: rephrase to indicate this as the main benefit.
* I've seen paranoia from both devs and non-devs about open source software opening them up to malicious contributions. They're afraid of it being like Wikipedia, where _anybody_ can modify things.
    * Suggestion: Also mention that most open source allows folks to propose changes to the project, perhaps?
* The Slides/Powerpoint analogy is very abstract and I don't understand it.
    * See https://www.codecademy.com/resources/docs/open-source and https://opensource.com/resources/what-open-source for good references of how to explain open source. I personally try to avoid analogies whenever possible, as they can be hard to understand and not everyone may know the content they build on.